### ISSUE TITLE
Configure the pencil icon

### ISSUE THREAD

**Issue by BlueCutOfficial** [DazzlingFugu/ember-fr-guides-source#180]
At the top right of each page of the guide, there's a pencil icon that is supposed to help people contribute by linking to the corresponding GitHub page.

These icons currently link to `https://github.com/ember-learn/guides-source`. We need to have them pointing to the `https://github.com/DazzlingFugu/ember-fr-guides-source` repository.

**Comment by BlueCutOfficial** [comment]
In `config/environment.js`, set:
```
guidemaker: {
  title: 'Ember Guides',
  sourceRepo: 'https://github.com/DazzlingFugu/ember-fr-guides-source'
},
```

This part works fine for redirecting to the right repo, but we still miss something to have the correct URL. Currently it contains the ember version instead of `release`. 

Possibly related to [this guidemaker-ember-template component](https://github.com/ember-learn/guidemaker-ember-template/blob/7ceefdc97b6caa95e9f9c350f413db6a07b29c3e/app/components/guides-article.js)?

**Comment by BlueCutOfficial** [comment]
Closed by #187
